pyiceberg/io/pyarrow.py

```diff
@@ -1339,7 +1339,10 @@ def update_min(self, val: Any) -> None:
     def update_max(self, val: Any) -> None:
         self.current_max = val if self.current_max is None else max(val, self.current_max)
 
-    def min_as_bytes(self) -> bytes:
+    def min_as_bytes(self) -> Optional[bytes]:
```

Comment:
Interesting, it is only affecting the bytes?

Reply:
It fails when we try to pack None values into certain structs like STRUCT_INT or STRUCT_LONG

```
>>> import struct
>>> struct.Struct("<i").unpack(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: a bytes-like object is required, not 'NoneType'
```

Reply:
Ah, I see. I didn't expand the code. Thanks!